projects/Mallard/src/hooks/use-edition-provider.tsx

```diff
@@ -116,6 +120,17 @@ export const fetchEditions = async (
     }
 }
 
+export const removeExpiredSpecialEditions = (
```

Comment:
We may need to remove the corresponding downloaded issue from the device as well. May be raise a separate ticket for that?

Reply:
Good point https://trello.com/c/e9O8wPMV/1558-delete-special-editions-after-they-have-expired